spec/requests/spree/checkout_controller_spec.rb

```diff
@@ -85,9 +85,9 @@
       let(:access_token) { 'accesstoken' }
       let(:payment) { create(:bolt_payment, amount: order.total, order: order) }
 
-      it 'calls the service to add addresses' do
+      it 'calls the job to add addresses' do
         user.addresses.each do |address|
-          expect(SolidusBolt::Accounts::AddAddressService).to have_received(:call).with(
+          expect(SolidusBolt::AddAddressJob).to have_received(:perform_later).with(
```

Comment:
I think we should actually test jobs like this

```rb
expect {
      # code to execute job
    }.to have_enqueued_job(SolidusBolt::AddAddressJob)
```

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/96/commits/0f5fef707383340c37526c114af66150cf238dbc